### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Addresses the issue: https://github.com/bacalhau-project/docs/issues/23

### CODE REVIEW

File: setting-up/workload-onboarding/container/index.md [hunk 1]
```diff
@@ -1,63 +1,32 @@
 ---
-description: How to use the Bacalhau Docker image
+description: How to use the Bacalhau CLI in Docker
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:2]: I'm not sure this adds clarity?

File: setting-up/workload-onboarding/container/index.md [hunk 2]
```diff
@@ -1,63 +1,32 @@
...
 ---
 
-# Bacalhau Docker Image
+# Bacalhau CLI from Docker
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:5]: It's not "from Docker" - it's a docker image?

File: setting-up/workload-onboarding/container/index.md [hunk 3]
```diff
@@ -1,63 +1,32 @@
...
 
-This documentation explains how to use the Bacalhau Docker image to run tasks and manage them using the Bacalhau client.
+This documentation explains how to use Bacalhau CLI from docker.
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:7]: The former i think is clearer.

File: setting-up/workload-onboarding/container/index.md [hunk 4]
```diff
@@ -1,63 +1,32 @@
...
 
 ## Prerequisites
 
-To get started, you need to install the Bacalhau client (see more information [here](../../../getting-started/installation.md)) and Docker.
+Install the [Bacalhau CLI in Docker](../../..//getting-started/installation#step-1.1-install-the-bacalhau-cli).
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:11]: Why do we need to install the bacalhau client at all? aren't we using the pre-built image?

File: setting-up/workload-onboarding/container/index.md [hunk 5]
```diff
@@ -1,63 +1,32 @@
...
-621f190d05c8: Pull complete
-Digest: sha256:3cda5619984de9b56c738c50f94188684170f54f7e417f8dcbe74ff8ec8eb434
-Status: Downloaded newer image for ghcr.io/bacalhau-project/bacalhau:latest
-ghcr.io/bacalhau-project/bacalhau:latest
-```
-
-You can also pull a specific version of the image, e.g.:
-
-```bash
-docker pull ghcr.io/bacalhau-project/bacalhau:v0.3.16
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:35]: why are we pinning to such an old version?

File: setting-up/workload-onboarding/container/index.md [hunk 6]
```diff
@@ -1,63 +1,32 @@
...
-
-To check the version of the Bacalhau client, run:
+## 1. Check the version of Bacalhau CLI
 
 ```bash
 docker run -t ghcr.io/bacalhau-project/bacalhau:latest version
 ```
 
-Expected Output:
+The output is similar to:
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:19]: This is "Expected Output" - the change is too wordy. If there are variables that will change, then just mention them.

File: setting-up/workload-onboarding/container/index.md [hunk 7]
```diff
@@ -1,63 +1,32 @@
...
 
 ```shell
 13:38:54.518 | INF pkg/repo/fs.go:81 > Initializing repo at '/root/.bacalhau' for environment 'production'
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:22]: this is horrible - @wdbaruni @frrist for your tracking. We should NEVER EVER print out raw go logs like this when starting. It is horribly confusing.

File: setting-up/workload-onboarding/container/index.md [hunk 8]
```diff
@@ -1,63 +1,32 @@
...
 CLIENT  SERVER  UPDATE MESSAGE
 v1.2.0  v1.2.0
 ```
 
-## 3. Running a Bacalhau Job
+## 2. Run a Bacalhau Job
 
-In the example below, an Ubuntu-based job runs to print the message 'Hello from Docker Bacalhau':
+For example to run an Ubuntu-based job that prints the message 'Hello from Docker Bacalhau':
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:29]: I like the former language better.

File: setting-up/workload-onboarding/container/index.md [hunk 9]
```diff
@@ -77,7 +46,7 @@ docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
 5. `--`: Separate Bacalhau parameters from the command to be executed inside the container
 6. `sh -c 'uname -a && echo "Hello from Docker Bacalhau!"'`: The command executed inside the container
 
-Let's have a look at the command execution in the terminal:
+The command execution in the terminal is similar to:
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:49]: The former feels friendlier.

File: setting-up/workload-onboarding/container/index.md [hunk 10]
```diff
@@ -86,27 +55,27 @@ ab95a5cc-e6b7-40f1-957d-596b02251a66
...
+To print the **content of the Job ID**, execute the following command:
 
 ```
 docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
     describe ab95a5cc-e6b7-40f1-957d-596b02251a66 \
         | grep -A 2 "stdout: |"
 ```
 
-Expected Output:
+The output is similar to:
```
Comment (aronchick) [inline setting-up/workload-onboarding/container/index.md:66]: Not "is similar to"

**Comment by Rajakavitha1** [comment]
Thank you for the suggestions @aronchick ! I will incorporate the changes as soon as possible.

File: setting-up/workload-onboarding/container/index.md (hunk 4 above)
Comment (Rajakavitha1) [reply setting-up/workload-onboarding/container/index.md:11]: I had to install bacalhau client else I kept getting the error: ```rkodhand@blr-mpe5j ~ % docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
    docker run \
        --id-only \
        --wait \
        ubuntu:latest \
        -- sh -c 'uname -a && echo "Hello from Docker Bacalhau!"'
12:02:36.828 | INF pkg/repo/fs.go:93 > Initializing repo at '/root/.bacalhau' for environment 'production'
publicapi: after posting request: "This endpoint is deprecated. See the migration guide at https://docs.bacalhau.org/references/cli-reference/command-migration for more information"```

File: setting-up/workload-onboarding/container/index.md (hunk 5 above)
Comment (Rajakavitha1) [reply setting-up/workload-onboarding/container/index.md:35]: Exactly that is why these instructions were removed from the guide and only the latest version is retained.

File: setting-up/workload-onboarding/container/index.md (hunk 3 above)
Comment (Rajakavitha1) [reply setting-up/workload-onboarding/container/index.md:7]: rewrote for making it more clear.

File: setting-up/workload-onboarding/container/index.md (hunk 6 above)
Comment (Rajakavitha1) [reply setting-up/workload-onboarding/container/index.md:19]: I just followed the standard documentation guidelines https://developers.google.com/style/code-in-text

File: setting-up/workload-onboarding/container/index.md (hunk 9 above)
Comment (Rajakavitha1) [reply setting-up/workload-onboarding/container/index.md:49]: Followed the word standard writing guidelines. https://developers.google.com/style/accessibility

File: setting-up/workload-onboarding/container/index.md [hunk 11]
```diff
@@ -70,77 +37,102 @@ docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
...
 ```shell
-docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
-    docker run \
+bacalhau docker run \                                   
         --id-only \
         --wait \
         --gpu 1 \
         ghcr.io/bacalhau-project/examples/stable-diffusion-gpu:0.0.1 -- \
             python main.py --o ./outputs --p "A Docker whale and a cod having a conversation about the state of the ocean"
 ```
```
Comment (Rajakavitha1) [inline setting-up/workload-onboarding/container/index.md:114]: This job was not successful for me and I kept getting the error:```
Error: not enough nodes to run job. requested: 1, available: 3, suitable: 0.
       • Node QmPLPUUj: node busy with available capacity {CPU: 0.3500000000000001, Memory: 3.7 GB, Disk: 1.7 TB, GPU: 0} and queue capacity {CPU: 1.5, Memory: 5.0 GB, Disk: 16 GB,
       GPU: 0}
       • Node QmbxGSsM: node busy with available capacity {CPU: 0.10000000000000009, Memory: 1.7 GB, Disk: 1.7 TB, GPU: 0} and queue capacity {CPU: 1, Memory: 2.0 GB, Disk: 2.8 kB,
       GPU: 0}
       • Node n-e002001e: node busy with available capacity {CPU: 0.6000000000000001, Memory: 2.7 GB, Disk: 1.7 TB, GPU: 0}```
       I am not sure if it was because of a miss at my end.

**Comment by Rajakavitha1** [comment]
Hi @aronchick  I tried the instructions and updated the guide.
Here's the terminal output.
[bacalhau.pdf](https://github.com/user-attachments/files/16922737/bacalhau.pdf)
